src/server/Monte.AuthServer/Services/UserService.cs

```diff
@@ -135,7 +138,7 @@ public async Task<Result> ChangePassword(string userId, string oldPassword, stri
         return new Result();
     }
 
-    public async Task<Result> GetUsers()
+    public async Task<Result<IEnumerable<UserDetails>>> GetUsers()
     {
         var users = await Task.Run(() => _userManager.Users);
```

Comment:
why

Reply:
needed a Task in method, changed now to Task.FromResult